src/App.js

```diff
@@ -57,13 +59,23 @@ class App extends Component {
   }
 
   sortMovies = category => {
-    this.setState({movies: sortMovies(this.state.movies, category)})
+    this.setState({movies: sortMovies(this.state.filtered, category)})
+  }
+
+  searchMovies = (input) => {
+    const term = input.toLowerCase()
+    this.setState({filtered: this.state.movies.filter(movie => {
+        const title = movie.title.toLowerCase()
```

Comment:
I like this function. It is a case-insnsitive search and filters through the full list and assigns it to the filtered list to render.